docs/05-technical-reference/00-custom-resources/evnt-01-subscription.md

```diff
@@ -9,30 +9,26 @@ The `subscriptions.eventing.kyma-project.io` CustomResourceDefinition (CRD) is a
 
 ## Sample custom resource
 
-This sample Subscription custom resource (CR) subscribes to an event called `sap.kyma.custom.commerce.order.created.v1`.
+This sample Subscription custom resource (CR) subscribes to an event called `order.created.v1`.
 
-> **WARNING:** Non-alphanumeric characters `[^a-zA-Z0-9]+` are not allowed in event names under the **spec.filter.filters.eventType.value** property. If any are detected, Eventing will remove them. Read [Event names](../evnt-01-event-names.md#event-name-cleanup) for more information.
+> **WARNING:** Prohibited characters in event names under the **spec.types** property, are not supported in some backends. If any are detected, Eventing may remove them. Read [Event names](../evnt-01-event-names.md#event-name-cleanup) for more information.
```

Comment:
I think it was good like this: `Eventing will remove`.

Reply:
Updated.